src/store/useLoggerStore.ts

```diff
@@ -15,7 +18,7 @@ const store: StateCreator<LoggerStore> = (set) => ({
 				updatedLogs.splice(maxLogs);
 			}
 
-			return { logs: updatedLogs };
+			return { logs: updatedLogs, isNewLogs: !state.isLoggerEnabled };
```

Comment:
Is that possible that `isNewLogs` will be false here or only true when we add new logs?

Reply:
It's possible to be true or false, depending on `isLoggerEnabled` . If (`isLoggerEnabled` = true) then `isNewLogs` = false . So, it will help not to show the `Badge` in the open System Logger